includes/class-wc-payments-features.php

```diff
@@ -14,19 +14,36 @@
  */
 class WC_Payments_Features {
 	const UPE_FLAG_NAME                     = '_wcpay_feature_upe';
+	const UPE_SPLIT_FLAG_NAME               = '_wcpay_feature_upe_split';
 	const WCPAY_SUBSCRIPTIONS_FLAG_NAME     = '_wcpay_feature_subscriptions';
 	const WOOPAY_EXPRESS_CHECKOUT_FLAG_NAME = '_wcpay_feature_woopay_express_checkout';
 	const AUTH_AND_CAPTURE_FLAG_NAME        = '_wcpay_feature_auth_and_capture';
 
 	/**
-	 * Checks whether the UPE gateway is enabled
+	 * Checks whether amy UPE gateway is enabled.
```

Comment:
sorry for this nitpick 
```suggestion
	 * Checks whether any UPE gateway is enabled.
```